src/DurableTask.SqlServer.AzureFunctions/SqlDurabilityOptions.cs

```diff
@@ -24,6 +24,8 @@ class SqlDurabilityOptions
         [JsonProperty("taskEventBatchSize")]
         public int TaskEventBatchSize { get; set; } = 10;
 
+        public ManagedIdentityOptions? ManagedIdentityOptions { get; set; }
```

Comment:
Let's add `[JsonProperty("managedIdentityOptions")]` to this new property.

Reply:
Unfortunately the Azure Functions configuration system doesn't like our use of nested options here. We'll need to flatten it in order for it to be recognized. My suggestion is that we add the three properties directly to the `SqlDurabilityOptions` class and rename them such that they all have the same prefix. For example, `azureManagedIdentityEnabled`, `azureManagedIdentityAuthorityHost`, and `azureManagedIdentityTenantId`.

Reply:
Sure, let's do it. Thanks for letting me know.